uint8 motor_id
uint8 direction
uint8 use_ramping
uint8 job_id
uint8 microstep

uint32 pulses
uint32 pulse_interval
uint32 pulse_on_period
uint32 ramping_steps
